packages/rule-schema-to-typescript-types/tests/index.test.ts

```diff
@@ -0,0 +1,148 @@
+import { schemaToTypes } from '../src/index';
+
+describe(schemaToTypes, () => {
+  it('returns a [] type when the schema is an empty array', () => {
+    const actual = schemaToTypes([]);
+
+    expect(actual).toMatchInlineSnapshot(`
+      "/** No options declared */
+      type Options = [];"
+    `);
+  });
+
+  it('returns a single Options type when the schema is not an array', () => {
+    const actual = schemaToTypes({ type: 'string' });
+
+    expect(actual).toMatchInlineSnapshot(`"type Options = string"`);
+  });
+
+  it('returns an array Options type when the schema is an array', () => {
+    const actual = schemaToTypes([{ type: 'string' }]);
+
+    expect(actual).toMatchInlineSnapshot(`"type Options = [string]"`);
+  });
+
+  it('returns a complex Options type when the schema contains an array of items', () => {
+    const actual = schemaToTypes([
+      {
+        type: 'array',
+        items: [{ type: 'string' }],
```

Comment:
I didn't notice the lint error: `perfectionist/sort-objects` expects properties to be sorted alphabetically. To fix:

```suggestion
        items: [{ type: 'string' }],
        type: 'array',
```